tests/address_bar_and_search/test_search_code_google_non_us.py

```diff
@@ -0,0 +1,68 @@
+import pytest
+from selenium.webdriver import Firefox
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions as ec
+
+from modules.browser_object import ContextMenu, Navigation
+from modules.page_object import AboutConfig
+
+
+# Set search region
+@pytest.fixture()
+def add_prefs():
+    return [
+        ("browser.search.region", "DE"),
+    ]
+
+
+def test_search_code_google_us(driver: Firefox):
+    """
+    C1365269 - Default Search Code: Google - non-US
+    This tests multiple ways of sending a search; Awesome bar,
+    Search bar and selected text
+    """
+
+    # Create objects
+    nav = Navigation(driver).open()
+    ac = AboutConfig(driver)
+    context_menu = ContextMenu(driver)
+
+    def search_code_assert():
+        # Function to check the search code of a Google search in US region
+        fx_code = "client=firefox-b-d"
+        nav.set_content_context()
+        search_url = driver.current_url
+        assert fx_code in search_url
+        nav.clear_awesome_bar()
+
+    # Check code generated from the Awesome bar search
+    nav.search("soccer")
+    nav.expect(ec.title_contains("Google Search"))
```

Comment:
its a stylistic thing but I notice everywhere else we use uppercase `ec` but it's up to you!